src/providers/bitcoin/BitcoinRPCProvider.js

```diff
@@ -14,7 +14,7 @@ export default class BitcoinRPCProvider extends JsonRpcProvider {
 
   async isAddressUsed (address) {
     address = String(address)
-    const utxo = await this.getUnspentTransactions(address)
+    const utxo = await this.getReceivedByAddress(address)
```

Comment:
something about this doesn't seem clear enough...
I'm looking at the return of `getreceivedbyaddress` RPC call, and it is the amount received by this address but to check this we do `utxo.length` as if the return would be a list of utxos. wouldn't the check be more like this?
```
const amoutnReceived = await this.getReceivedByAddress(address)
return amountReceived > 0
```

Reply:
@monokh good catch. Updated with proper logic.